css/style.css

```diff
@@ -0,0 +1,3958 @@
+}
+
+.testimonials-card-details {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: end;
+  -ms-flex-align: end;
+  align-items: flex-end;
+}
+
+.testimonials-card-name {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-column-gap: 16px;
+  -moz-column-gap: 16px;
+  column-gap: 16px;
+}
+.testimonials-card-title {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 1;
+  overflow: hidden;
+}
+
+.testimonials-rating::after {
+  content: "";
+  display: block;
+  margin-top: 6px;
+  max-width: 80px;
+  height: 16px;
+  -webkit-mask-image: url("../img/icons/icon-star.svg");
+  mask-image: url("../img/icons/icon-star.svg");
+  background-color: var(--accent-color);
+  -webkit-mask-size: 16px;
+  mask-size: 16px;
+  -webkit-mask-repeat: repeat-x;
+  mask-repeat: repeat-x;
+}
+
+.testimonials-btn-group {
+  margin-left: auto;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-column-gap: 8px;
+  -moz-column-gap: 8px;
+  column-gap: 8px;
+}
+
+.btn-arrow {
+  padding: 10px;
+  width: 44px;
+  height: 44px;
+  border-radius: 50%;
+  border: none;
+  background-color: var(--bg-color);
+  fill: var(--accent-color);
+  cursor: pointer;
+  -webkit-transition: all var(--trasition-effect);
+  -o-transition: all var(--trasition-effect);
+  transition: all var(--trasition-effect);
+}
+
+.arrow-less {
+  margin-top: 4px;
+  margin-right: 4px;
+  rotate: 90deg;
+}
+
+.arrow-greater {
+  margin-top: 4px;
+  margin-left: 4px;
+  rotate: -90deg;
+}
+
+.btn-arrow:hover,
+.btn-arrow:active,
+.btn-arrow:focus {
+  fill: var(--secondary-color);
+  background-color: var(--accent-color);
+}
+
+/*************** Pricing section *******************/
+
+.pricing .section-name,
+.pricing .section-title {
+  text-align: center;
+  margin-inline: auto;
+  max-width: 566px;
+}
+
+.membership-plan-list {
+  margin-top: 48px;
+  padding-top: 32px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-column-gap: 40px;
+  -moz-column-gap: 40px;
+  column-gap: 40px;
+}
+
+.membership-plan-item {
+  max-width: 400px;
+  width: 100%;
+}
+
+.membership-plan {
+  padding: 32px;
+  border-radius: var(--border-radius-m);
+  -webkit-box-shadow: var(--box-shadow-m);
+  box-shadow: var(--box-shadow-m);
+}
+
+.membership-plan-title {
+  margin-bottom: 16px;
+  text-transform: capitalize;
+}
+
+.membership-plan-price {
+  margin-bottom: 32px;
+  color: var(--opacity-50);
+  text-align: left;
+}
+.membership-plan-number {
+  font-family: var(--font-header);
+  font-weight: var(--font-bold);
+  font-size: 48px;
+  line-height: 1.2;
+  color: var(--primary-color);
+  overflow: hidden;
```

Comment:
I don’t quite understand why you need `overflow: hidden` here

Reply:
#Fixed